tests/spec/core/show-people-spec.js

```diff
@@ -0,0 +1,240 @@
+import { html } from "../../../src/core/import-maps.js";
```

Comment:
Maybe we can move this file to unit tests folder (`tests/unit/core/show-people-spec.js`)?

Reply:
Sure.